slices/rsyslog.yaml

```
@@ -0,0 +1,46 @@
+package: rsyslog
+
+essential:
+  - rsyslog_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libestr0_libs
+      - libfastjson4_libs
+      - libsystemd0_libs
+      - libuuid1_libs
+      - libzstd1_libs
+      - rsyslog_config
+      - rsyslog_data
+      - zlib1g_libs
```

```suggestion
      - libc6_libs
      - libestr0_libs
      - libfastjson4_libs
      - libsystemd0_libs
      - libuuid1_libs
      - rsyslog_config
      - rsyslog_data
      - zlib1g_libs
```
`libzstd1` is not a direct dependent library of this binary file. It should however be listed as an essential in the `_libs` slice.

---

Thanks for pointing that out. Btw how do I know that which dependency libs are used by `rsyslog` binary?

---

You can run `ldd /usr/sbin/rsyslogd` command to verify the libraries linked to it.